src/core/Form/Checkbox/Checkbox.baseStyles.tsx

```diff
@@ -85,7 +82,7 @@ const largeVariantStyles = ({ theme }: SuomifiThemeProp) => css`
         top: 4px;
       }
     }
-    & .fi-checkbox_hintText {
+    & .fi-hint-text {
```

Comment:
These are only for Checkbox? Not sure if it matters, but this implementation relies on internal structure of hint text rather than defining the padding on the container of the element. The components are of course closely coupled anyways, but this would probably be an easy change? If it requires hacky solutions then I'd also prefer using this one.

Reply:
It could be done like this, **IF**, the status text wouldn't have different padding (0px) than the label and hint.

```
    & > * {
      padding-left: ${theme.spacing.xxl};
    }
```

I don't know other ways to add the style without specifying different style for hint/status text element, like the current implementation.

Reply:
Ah, my bad. This is only to apply outer margins / padding to the component. As proposed earlier, maybe we should change the styles of hint text and leave the outer padding / margin to wrapping components like done here.

Reply:
Decided to fix these with another issue / branch.